Makefile

```diff
@@ -1,15 +1,83 @@
-#
-# Just a minimal Makefile for now to install the most basic components
-#
-# Currently installs the assembler and the app loader library
-#
+include $(BUILD_DIR)/makefile.d/base.mk
+
+ROOTDIR = .
+TARGET = libpru-driver
+CROSS_COMPILE?=arm-linux-gnueabihf-
```

Comment:
Yeah, that's what I figured. To which I can point out: https://github.com/Vorne/pru-driver/blob/master/pru_sw/app_loader/interface/Makefile#L3

Not that I don't hate spaces, but this one **happened** to be copy/paste-ish.